src/coordinate3d.hpp

```diff
@@ -89,6 +89,92 @@ struct Coordinate3D {
      * @return int16_t
      */
     int16_t getZ();
+
+    /**
+     * @brief coordinate3D += operator
+     *
+     * Returns a coordinate3D after adding another coordinate3D to it.
+     *
+     * @param rhs : const Coordinate3D &
+     *
```

Comment:
According to the Wiki, this newline shouldn't be here, although I won't reject it here.